docker/runtime/ruby/Makefile

```diff
@@ -1,11 +1,25 @@
 
+build2.3:
+	env GOOS=linux GOARCH=amd64 go build $$GOPATH/src/github.com/kubeless/kubeless/pkg/function-proxy/proxy.go
+	docker build -t kubeless/ruby:2.3$$RUNTIME_TAG_MODIFIER -f Dockerfile .
```

Comment:
you need to change the name of the Dockerfiles here (`Dockerfile.2.3`, `Dockerfile.2.4`, `Dockerfile.2.5`)

Reply:
Yes, of course, those Dockerfile refences slipped my attention, thanks for point this out @andresmgot